fonts.css

```diff
@@ -1,18 +1,26 @@
 @font-face {
   font-family: 'Museo';
-  src: url('../fonts/MuseoSansCyrl-500.woff')format('woff'), url('../fonts/MuseoSansCyrl-500.woff2')format('woff2');
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)